DESIGN-README.md

```diff
@@ -0,0 +1,37 @@
+# Getting Started
+
+## Design Tools
+Design tools we use:
+* Sketch
+* Invision
+
+## Design Files
+* Sketch file (request link)
+* [Icon Set](https://www.emojione.com/developers/download?key=8YNLFBYZAB47ST8ZDZV3)
+
+## Styleguide/Branding
+This application was designed using an Atomic design system.
+
+* [Styleguide](http://demo.getcalfresh.org/styleguide)
+* [Styleguide code](https://github.com/codeforamerica/gcf-backend/tree/master/app/views/styleguides)
```

Comment:
this is a private repo -- are we worried about linking to it from a public repo?

Reply:
@alanjosephwilliams Are we concerned about this? Should i remove the link to the private repo?

Reply:
per alan in standup, this is NOT a problem